src/nvim/version.c

```diff
@@ -80,8 +80,8 @@ static int included_patches[] = {
   784,
   //783,
   //782,
-  //781,
-  //780,
+  781,
+  //780 NA
```

Comment:
Ok, if you have time, I have a list about this, and there will be ten now~:
//777 NA Readme  https://code.google.com/p/vim/source/detail?r=v7-4-777
//772 NA makefile related and if_* file Readme  https://code.google.com/p/vim/source/detail?r=v7-4-772
//766 NA term.c Readme  https://code.google.com/p/vim/source/detail?r=v7-4-766
//763 NA if_lua.c   https://code.google.com/p/vim/source/detail?r=v7-4-763
//762 NA term.c  https://code.google.com/p/vim/source/detail?r=v7-4-762
//759 NA if_lua.c https://code.google.com/p/vim/source/detail?r=v7-4-759
//756 NA if_perl.xs https://code.google.com/p/vim/source/detail?r=v7-4-756
//750 NA makefile https://code.google.com/p/vim/source/detail?r=v7-4-750
//445 NA clipboard relate  https://github.com/neovim/neovim/pull/2274/ https://code.google.com/p/vim/source/detail?r=v7-4-750